ldap/servers/plugins/replication/repl-session-plugin.h

```diff
@@ -21,16 +21,16 @@
  * The callback can allocate some private data to return.  If so
  * the callback must define a repl_session_plugin_destroy_agmt_cb
  * so that the private data can be freed.  This private data is passed
- * to other callback functions on a master as the void *cookie argument.
+ * to other callback functions on a supplier as the void *cookie argument.
  */
 typedef void *(*repl_session_plugin_agmt_init_cb)(const Slapi_DN *repl_subtree);
 #define REPL_SESSION_PLUGIN_AGMT_INIT_CB 1
 
 /*
  * Callbacks called when acquiring a replica
  *
- * The pre and post callbacks are called on the sending (master) side.
- * The receive and reply callbacks are called on the receiving (replica)
+ * The pre and post callbacks are called on the sending (supplier/provider)
```

Comment:
I think it was agreed upon that we'll use sender / receiver for the replication session.
(and supplier / hub / consumer for replica role)
Partially because we want to avoid the confusion around two different concepts.

Or do I miss something?